docs/extensions/guides/stores.md

```diff
@@ -42,18 +42,17 @@ export class ExamplePreferencesStore extends Store.ExtensionStore<ExamplePrefere
         enabled: false
       }
     });
+    makeObservable(this);
   }
 
   protected fromStore({ enabled }: ExamplePreferencesModel): void {
     this.enabled = enabled;
   }
 
   toJSON(): ExamplePreferencesModel {
-    return toJS({
+    return {
       enabled: this.enabled
-    }, {
-      recurseEverything: true
```

Comment:
@stefcameron it's removed from current docs. Is that what you asked? ^

Reply:
If by "current" you mean the docs associated with Lens 5 changes, then yes. If by "current" you mean Lens 4.2.4 docs, then it's still there (and should be, I guess, since 4.2.4 still users `mobx` < 6).

Reply:
By "current" I mean version mentioned in `package.json` or above whenever it's gonna be released.
Otherwise in current branch all mentioning about `recurseEverything:true` was removed.

Reply:
Ok, thank you! 👍